src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy.mdx

```diff
@@ -136,3 +136,134 @@ The policies index lists policies in alphabetical order. To view or search for e
 <Callout variant="tip">
   To view [policy and condition information for a specific entity](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/view-policy-conditions-new-relic-products), select the **Settings > Alert conditions** page from the entity's product UI.
 </Callout>
+
+## Policy maintenance quick reference [#quick-reference-policies]
+
+After you create a policy, you can:
+
+* Add more conditions to the policy.
+* Edit the conditions' names, targets, or thresholds.
+* Disable conditions.
+* Continue the policy setup process by adding one or more notification channels to it.
```

Comment:
Can we just say "Add one or more notification channels to a policy."

Reply:
Yes, excellent point.